test/Microsoft.Health.Fhir.R4.Tests.Integration/Persistence/FhirStorageVersioningPolicyTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿// -------------------------------------------------------------------------------------------------
```

Comment:
What are your thoughts on using the `#if Stu3` method to combine the unit tests into one file?

Reply:
Ah, I really like the idea of this. Just tried it out locally, but the `#if Stu3` flag doesn't seem to get picked up by the integration tests. I noticed all other tests that use this are either unit tests or e2e tests. @brendankowitz is this by-design?

Reply:
Going to go ahead and merge the PR for now, since I don't think this should be blocking. However, I am happy to add this in later if it makes sense.